specification/synapse/data-plane/Microsoft.Synapse/preview/2020-08-01-preview/checkAccessSynapseRbac.json

```diff
@@ -0,0 +1,246 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-08-01-preview",
+    "title": "SynapseClient"

```

Comment:
Let's change `title` in all Swagger files as `AccessControlClient`. `SynapseClient` is too general